components/VFBMain.js

```diff
@@ -792,12 +781,6 @@ export default class VFBMain extends React.Component {
         }
         this.setState({ UIUpdated: false });
         break;
-      case 'graphVisible':
```

Comment:
why are we removing this? should not we follow the same pattern to handle the check for visibility for the components created by the factory method (flexlayout)?

Reply:
Switch case has been added back.